v3-sdk/collecting-fees/src/example/Example.tsx

```diff
@@ -79,6 +79,19 @@ const Example = () => {
     setTxState(await collectFees(position))
   }, [])
 
+  // Formatted Data
+
+  let positionInfoStrings: string[] = []
```

Comment:
We should make this a `useMemo` to avoid needed rerenders - same as `useCallback` above but for variables (as opposed to functions)